web/src/modules/topic/components/Score/ScorePie.tsx

```diff
@@ -30,28 +43,33 @@ export const ScorePie = ({ pieDiameter, arguableId, arguableType }: Props) => {
   });
 
   return (
-    <Box display="flex" width={pieDiameter} height={pieDiameter}>
+    <StyledBox display="flex" width={pieDiameter} height={pieDiameter}>
       <PieChart
         data={data}
-        radius={50} // is relative to viewbox size [100, 100]
+        radius={45} // is relative to viewbox size [100, 100]
         label={({ dataEntry }) => dataEntry.key}
         labelStyle={() => ({
           fontSize: "14", // is relative to viewbox size [100, 100]
           pointerEvents: "none",
+          webKitUserSelect: "none" /* Safari */,
+          msUserSelect: "none" /* IE 10 and IE 11 */,
```

Comment:
yooooo idk, mozilla doesn't even show IE in their browser compatibility list anymore, which makes me think it's not worth supporting (we'd have to go elsewhere for compatibility docs i guess?). what do you think?

Reply:
Its a habit for me to put other browser CSS along with the default. I can remove it if you want! (I have no preference)